### ISSUE TITLE
[Bug] It is not possible to run this application on your computer. Windows 10 x86 . 

### ISSUE THREAD

**Issue by Dev65432** [AppFlowy-IO/AppFlowy#5452]
### Bug Description

When I start the program, I get:: 
```
It is not possible to run this application on your computer.
To find the version of your computer, contact the publisher of the application.
```

Used: Windows 10 x86 10.0.17763 Build 17763

### How to Reproduce

I have completed:
    - I downloaded `AppFlowy 0.5.7-05/10/2024` | [AppFlowy/releases | 0.5.7-05/10/2024](https://github.com/AppFlowy-IO/AppFlowy/releases/tag/0.5.7)
    - I have installed;
    - I'm launching `c:\Program Files\AppFlowy\AppFlowy.exe`;
    - Result: I receive a message
```
It is not possible to run this application on your computer.
To find the version of your computer, contact the publisher of the application.
```

### Expected Behavior

I have completed:
    - I'm launching `c:\Program Files\AppFlowy\AppFlowy.exe`;
    - Result: it starts **AppFlowy.exe**.

### Operating System

Windows 10 x86 10.0.17763 Build 17763

### AppFlowy Version(s)
0.5.7-05/10/2024  | [AppFlowy/releases | 0.5.7-05/10/2024 ](https://github.com/AppFlowy-IO/AppFlowy/releases/tag/0.5.7)

### Screenshots

![image](https://github.com/AppFlowy-IO/AppFlowy/assets/171281745/0e89e1ff-5819-4716-9bdc-1ce0aa3d70bc)

The text in the picture:
```
It is not possible to run this application on your computer.
To find the version of your computer, contact the publisher of the application.
```

**Comment by LucasXu0** [comment]
Hi, @Dev65432. What's your CPU type?

**Comment by Dev65432** [comment]
@LucasXu0 

<html xmlns:v="urn:schemas-microsoft-com:vml"
xmlns:o="urn:schemas-microsoft-com:office:office"
xmlns:x="urn:schemas-microsoft-com:office:excel"
xmlns="http://www.w3.org/TR/REC-html40">

<head>

<meta name=ProgId content=Excel.Sheet>
<meta name=Generator content="Microsoft Excel 15">
<link id=Main-File rel=Main-File
href="file:///C:\Users\USER00~1\AppData\Local\Temp\msohtmlclip1\01\clip.htm">
<link rel=File-List
href="file:///C:\Users\USER00~1\AppData\Local\Temp\msohtmlclip1\01\clip_filelist.xml">
<style>

</style>
</head>

<body link="#0563C1" vlink="#954F72">


OS Name | Microsoft Windows 10 Enterprise LTSC
-- | --
Version | 10.0.17763 Build 17763
Additional description of the OS | Unavailable
OS Manufacturer | Microsoft Corporation
System Name | PC0001
Manufacturer | ASUSTeK Computer Inc.
Model | X71SL
Type | An x86-based computer
of SKU system | To Be Filled By O.E.M.
Processor | Intel(R) Core(TM)2 Duo CPU P7350 @ 2.00GHz, 2000 MHz, Cores: 2, Logical   processors: 2
BIOS Version | American Megatrends Inc. 203, 09/08/2008
SMBIOS version | 02.May
Version of the built-in controller | 255.255
BIOS Mode | Outdated
The manufacturer of the main board | PEGATRON CORPORATION
The model of the main board | X71SL
Main Board version | 1.0
The role of the platform | Mobile
Secure Boot Status | Not supported
PCR7 Configuration | Binding is not possible
Windows Folder | C:\Windows
System folder | C:\Windows\system32
Download device | \Device\HarddiskVolume2
The language of the system | Russia
Hardware-dependent Layer (HAL) | Version = "10.0.17763.1971"
User Name | PC0001\User00001
Time zone | RTZ 2 (winter)
Installed RAM (RAM) | 3.00 GB
The full amount of physical memory | 3.00 GB
Available physical memory | 653 MB
Total virtual memory | 7.26 GB
Virtual memory available | 1.55 GB
The size of the swap file | 4.26 GB
The swap file | C:\pagefile.sys
Core DMA protection | Unavailable
Virtualization-based security | Not included
Hyper-V — extensions of the virtual   machine monitoring mode | No
Hyper-V extensions for second—level   address translation | No
Hyper-V virtualization is enabled in the   firmware | No
Hyper-V — Data execution prevention | Yes



</body>

</html>

**Comment by annieappflowy** [comment]
Sadly, Flutter doesn't support 32 bit anymore, so there isn't a way for us to make appflowy runnable on your machine.
